src/wallet/KintoWalletFactory.sol

```diff
@@ -204,6 +204,12 @@ contract KintoWalletFactory is Initializable, UUPSUpgradeable, OwnableUpgradeabl
         IFaucet(_faucet).claimKintoETH(_signatureData);
     }
 
+    function sendMoneyToAccount(address target) external payable override {
```

Comment:
```suggestion
    function transfer(address target) external payable override {
```

Reply:
dont want to change public methods until later